enhanced_rag/azure_integration/automation/reindex_manager.py

```
@@ -224,7 +255,13 @@
             # Remove backup metadata
             schema.pop('_backup_metadata', None)
             
-            success = await self.reindex_ops.drop_and_rebuild(backup_path)
+            # Recreate index with backed-up schema
+            try:
+                await self.ops.delete_index(self._index_default_name)
+            except Exception:
+                pass
```

Ruff (0.12.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)